ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -66,6 +66,33 @@ def test_tensorflow_Acosh(  # NOQA
     )
 
 
+# ApproximateEqual
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.ApproximateEqual",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_ApproximateEqual(  # NOQA
+    *,
+    dtype_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        x=x[0],
```

Comment:
please also add the arguments `y` and `tolerance`, hints on data generation will be added as another comment